js/classes/game_engine.js

```diff
@@ -204,14 +222,14 @@ class GameEngine {
         const minY = DIMENSION.MIN_Y + 18;
         let gameY = minY;
         for (let blockX = DIMENSION.MIN_BLOCK_X; blockX <= DIMENSION.MAX_BLOCK_X; blockX += 5) {
-            for (let blockY = DIMENSION.MIN_BLOCK_Y ; blockY <= DIMENSION.MAX_BLOCK_Y; blockY += 5) {
+            for (let blockY = DIMENSION.MIN_BLOCK_Y; blockY <= DIMENSION.MAX_BLOCK_Y; blockY += 5) {
                 let pt = "(" + blockX + ", " + blockY + ")";
-                CTX.fillText(pt, gameX - CAMERA.x, gameY - CAMERA.y, Block.SCALED_SIZE);
+                CTX.fillText(pt, gameX - CAMERA.pos.x, gameY - CAMERA.pos.y, Block.SCALED_SIZE);
```

Comment:
Guessing this is also for the grid @B1GD3VD0G ? To display the coordinates of the blocks?